src/services/blt_rewards.py

```
@@ -0,0 +1,793 @@
+    comment += "\n---\n"
+    comment += (
+        f"**Scoring this month** (across {owner} org): Open PRs (+1 each), Merged PRs (+10), "
+        "Closed (not merged) (−2), Reviews (+5; first two per PR in-month), "
+        "Comments (+2, excludes CodeRabbit). Run `/leaderboard` on any issue or PR to see your rank!\n"
+    )
+    if note:
+        comment += f"\n> Note: {note}\n"
+    return comment
+
+
+def format_reviewer_leaderboard_comment(leaderboard_data: dict, owner: str, pr_reviewers: list = None) -> str:
```

Ruff (0.15.7)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)